catalog/admin.py

```diff
@@ -1,3 +1,32 @@
 from django.contrib import admin
+from .models import Author, Genre, Book, BookInstance
 
-# Register your models here.
+
+class AuthorAdmin(admin.ModelAdmin):
+    list_display = ("last_name", "first_name", "date_of_birth", "date_of_death")
+    fields = ["first_name", "last_name", ("date_of_birth", "date_of_death")]
+
+
+admin.site.register(Author, AuthorAdmin)
```

Comment:
nên đồng nhất cách đăng ký
```suggestion
@admin.register(Author)
class AuthorAdmin(admin.ModelAdmin):
```